### PR TITLE
MCKIN-24383 & 24384 VB for GP v2 and Adventure

### PR DESCRIPTION
**Pull request by musmanmalik**
VB for GP v2 and Adventure

### CODE REVIEW

**Comment by ihtram** [comment]
@musmanmalik could you please share the xblock-adventure PR as well?

**Comment by musmanmalik** [comment]
@ihtram https://github.com/edx-solutions/xblock-adventure/pull/26

File: requirements/edx/custom.txt [hunk 1]
```diff
@@ -11,15 +11,15 @@
 -e git+https://github.com/open-craft/xblock-drag-and-drop-v2.git@82c9dc5e16d10793e8b79e60661e1a78893fce25#egg=xblock-drag-and-drop-v2-new
 -e git+https://github.com/edx-solutions/xblock-ooyala.git@v4.0.7#egg=xblock-ooyala==4.0.7
 git+https://github.com/edx-solutions/xblock-group-project.git@0.1.3#egg=xblock-group-project==0.1.3
--e git+https://github.com/edx-solutions/xblock-adventure.git@0.2#egg=xblock-adventure==0.2
+-e git+https://github.com/edx-solutions/xblock-adventure.git@0.2#egg=xblock-adventure==0.4
```
Comment (ihtram) [inline requirements/edx/custom.txt:14]: @musmanmalik don't we need to replace xblock-adventure.git@0.2 with xblock-adventure.git@0.4?

**Review by ihtram — CHANGES_REQUESTED** [review]

File: requirements/edx/custom.txt [hunk 2]
```diff
@@ -11,15 +11,15 @@
...
 -e git+https://github.com/open-craft/xblock-poll.git@v1.10.1#egg=xblock-poll==1.10.1
 -e git+https://github.com/open-craft/problem-builder.git@v3.5.7#egg=xblock-problem-builder==3.5.7
 -e git+https://github.com/OfficeDev/xblock-officemix.git@86238f5968a08db005717dbddc346808f1ed3716#egg=xblock-officemix
 -e git+https://github.com/open-craft/xblock-chat.git@v0.2.8#egg=xblock-chat==0.2.8
 -e git+https://github.com/open-craft/xblock-eoc-journal.git@v0.9.1#egg=xblock-eoc-journal==0.9.1
 -e git+https://github.com/mckinseyacademy/xblock-scorm.git@v2.2#egg=xblock-scorm==2.2
 -e git+https://github.com/mckinseyacademy/xblock-diagnosticfeedback.git@v0.2.5#egg=xblock-diagnostic-feedback==0.2.5
--e git+https://github.com/open-craft/xblock-group-project-v2.git@0.9.1#egg=xblock-group-project-v2==0.9.1
+-e git+https://github.com/open-craft/xblock-group-project-v2.git@0.9.1#egg=xblock-group-project-v2==0.10.0
```
Comment (ihtram) [inline requirements/edx/custom.txt:22]: same goes here. replace 0.9.1 with 0.10.0

**Review by ihtram — CHANGES_REQUESTED** [review]

File: requirements/edx/custom.txt (hunk 1 above)
Comment (musmanmalik) [reply requirements/edx/custom.txt]: Thanks for Pointing.

**Comment by musmanmalik** [comment]
@ihtram Please proceed. We need this on QA as soon as possible.

**Review by ihtram — APPROVED** [review]
👍

**Pull request merged**